### ISSUE TITLE
App Store Connect rejecting application due to "UIWebView" usage

### ISSUE THREAD

**Issue by riccardominato** [JimmyPun610/BarcodeScanner.XF#87]
Hello,

thank you for your useful library.

I'm facing an issue trying to send my Xamarin Forms application to TestFlight.
The message I receive states:

> ITMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted. Instead, use WKWebView for improved security and reliability.

I'm using the latest stable version of BarcodeScanner.XF (5.0.9) and Xamarin Forms is updated to 5.0.0.2125.
The iOS configuration in AppDelegate.cs is equal to what is reported in the readme.

I'm sure the issue is caused by BarcodeScanner.XF because I tried to remove it and the upload terminated with success.

I read, searching for a solution, that it could be caused by Firebase, but I have not idea on how to intervene, if this is the case.

Thank you in advance.

**Comment by stepkillah** [comment]
have you tried add mtouch arguments?
https://stackoverflow.com/a/61701124

**Comment by riccardominato** [comment]
I tried with `--optimize=experimental-xforms-product-type`, but I didn't found `--optimize=force-rejected-types-removal` in my research.

I ended up using [AVCamBarcode](https://docs.microsoft.com/en-us/samples/xamarin/ios-samples/ios11-avcambarcode/) for the iOS project and keeping an approach similar to the one used by BarcodeScanner.XF for Android.
That doesn't require any mtouch argument.

Thank you for your reply.
